display/d.labels/do_labels.c

```diff
@@ -67,7 +71,15 @@ int initialize_options(void)
 
 int do_labels(FILE *infile, int do_rotation)
 {
-    char buff[128];
+    char buff[BUFFSIZE];
+    char buff_fmt[10];
+    char font_fmt[10];
+    char word_fmt[10];
```

Comment:
To avoid using "magic numbers" you may (re-)use `WORDSIZE` here.

To enable the scope of `word_fmt` to `scan_ref()`, you should declare the variable as `static char` for file scope around line 40. (For symmetry, you may do the same for buff_fmt and font_fmt too).

Reply:
done

Reply:
You may change these to`char buff_fmt[WORDSIZE];` etc. The `10` would have been enough, but to avoid another magic number that would be sufficient.

Reply:
done all 3 of them changed to WORDSIZE